cypress/integration/06.WritePost.feature

```diff
@@ -17,7 +17,7 @@ Feature: Create a post
     for active citizenship.
     """
     And I click on "Save"
-    Then I get redirected to "/post/my-first-post/"
+    Then I get redirected to ".../my-first-post"
```

Comment:
> [<img alt="mattwr18" height="40" width="40" align="left" src="https://avatars1.githubusercontent.com/u/38261864?s=88&v=4">](https://github.com/mattwr18) **Authored by [mattwr18](https://github.com/mattwr18)**
_<time datetime="2019-03-26T23:00:49Z" title="Wednesday, March 27th 2019, 12:00:49 am +01:00">Mar 27, 2019</time>_

---
> **Outdated (history rewrite)** - original diff
---
```diff
@@ -17,7 +17,7 @@ Feature: Create a post
     for active citizenship.
     """
     And I click on "Save"
-    Then I get redirected to "/post/my-first-post/"
+    Then I get redirected to ".../my-first-post"
```


It's not so clear how this works until one checks out https://github.com/Human-Connection/Human-Connection/pull/248/files#diff-cc51d55abc8d6a1b33fd1f8885365953R227, but maybe there is nothing one can do about that